test/issues/general/test_4165.test

```diff
@@ -0,0 +1,31 @@
+# name: test/issues/general/test_4165.test
+# description: Issue 4165: SIGSEGV on Debian Buster amd64
+# group: [general]
+
+statement ok
```

Comment:
Perhaps set the seed so the test is deterministic? Otherwise it could accidentally not trigger the failure case. Not likely, but you never know...

Reply:
Good idea, done